lib/manager/npm/extract/index.ts

```diff
@@ -181,6 +182,25 @@ export async function extractPackageFile(
       }
       return dep;
     }
+
+    // support for volta
+    if (depType === 'volta') {
+      if (depName === 'node') {
+        dep.datasource = 'github';
+        dep.lookupName = 'nodejs/node';
+        dep.versionScheme = 'node';
+      } else if (depName === 'yarn') {
+        dep.datasource = 'npm';
+        dep.commitMessageTopic = 'Yarn';
+      } else {
+        dep.skipReason = 'unknown-volta';
+      }
+      if (!isValid(dep.currentValue)) {
```

Comment:
What happens if they specify a version like `1.10` for yarn?

Reply:
It updates it to 1.17

Reply:
You've tested this working? I wasn't sure if we considered 1.10 to be a valid value

Reply:
Yes, tested it against my test repo for volta.

![Screen Shot 2019-09-23 at 8 51 59 AM](https://user-images.githubusercontent.com/30690618/65402251-d78df000-ddea-11e9-84fa-e90e21962aa1.png)